src/Controls/src/Core/Compatibility/Handlers/TableView/Android/TableViewRenderer.cs

```diff
@@ -126,6 +126,21 @@ public override SizeRequest GetDesiredSize(double widthConstraint, double height
 			return base.GetDesiredSize(widthConstraint, heightConstraint);
 		}
 
+		protected override void OnMeasure(int widthMeasureSpec, int heightMeasureSpec)
```

Comment:
How did you reach the conclusion of this fix?

I think there's still some issues around our fix on this one. So, I'm curious to pick your brain how you got here.

Reply:
Actually the idea for this fix was initially posted by @Bennynation at https://github.com/dotnet/maui/issues/5924#issuecomment-1854000967. Maybe he can tell us how he got there? (I only took his sketch and molded it into a PR.)

Reply:
I think I finally have a pretty good idea. 

The problem stems from this line here
https://cs.android.com/android/platform/superproject/main/+/main:frameworks/base/core/java/android/widget/ListView.java;l=1321

When you have an AT_MOST measure, the listview will basically create a scrap copy of itself to measure which confuses the whole xplat system. Because it doesn't know that the platform is just creating an in memory listview to get a height measurement. The code here basically bypasses all of that. 

Now just have to figure out if this solution makes things worse, better, or if the goldilocks version.